L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -1089,7 +1085,7 @@ namespace trklet {
     //Overlap size for the overlap phi bins in DR
     double phiOverlapSize_{M_PI / 360};
     //The maximum number of tracks that are compared to all the other tracks per rinv bin
-    int numTracksComparedPerBin_{9999};
+    int numTracksComparedPerBin_{32};
```

Comment:
Original code allowed 9999 tracks compared, and you reduced this to 32. Does this by itself explain the drop in tracking efficiency that causes git CI to fail for HYBRID?

Reply:
It's definitely possible, but we can't leave it at 9999, can we?

Reply:
I forgot to change this when I switched to merging. I locally ran the merging version of the code (only HYBRID) with 1000 events over the same sample as the CI, and it doesn't seem like anything was affected from switching to 32 CM.